.gitignore

```diff
@@ -1 +1,25 @@
-**/.DS_Store
```

Comment:
Glad to have this, it may be worth deleting the currently pushed xcode configs in the future